src/main/java/com/lobo/onThisDay/model/Person.java

```diff
@@ -0,0 +1,22 @@
+package com.lobo.onThisDay.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ *
+ * Model para representar pessoas, de inicio esse model terá muita influecia da feature de consulta de nascimentos.
+ *
+ * @author Gustavo Lobo
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Person {
```

Comment:
Se isso for um DTO, sugiro deixar isso claro no nome da classe, algo como:
PersonDTO

Reply:
Vou alterar para ser um DTO, no entando a ideia é que haja uma classe futuramente, que consista numa entity, já que o plano para a API é não perder os dados coletados e que a gente persista eles num banco.